images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -222,7 +222,9 @@ function Get-NewmanVersion {
 }
 
 function Get-GHVersion {
-    return "GitHub CLI $(gh --version)"
+    $(gh --version) -match "gh version (?<version>.+)" | Out-Null
+    $ghVersion = $Matches.Version
+    return "GitHub CLI $ghVersion"
```

Comment:
The GitHub CLI is incorrect display in the documentation - https://github.visualstudio.com/virtual-environments/_build/results?buildId=79013&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae string 92

![image](https://user-images.githubusercontent.com/47745270/91143472-42ab0b00-e6bb-11ea-9181-2607c0cb1777.png)

Reply:
thank you for build output, fixed the issue with regexp